tests/acceptance/repo/settings-test.js

```diff
@@ -118,13 +118,13 @@ test('view settings', function (assert) {
     assert.equal(settingsPage.environmentVariables(1).value, '••••••••••••••••');
 
     assert.equal(settingsPage.crons(0).branchName, 'daily-branch');
-    // assert.ok(settingsPage.crons(0).lastRun.indexOf('Last:') === 0, 'Shows last run time of cron build');
-    // assert.ok(settingsPage.crons(0).nextRun.indexOf('Next:') === 0, 'Shows next run time of cron build');
+    assert.ok(settingsPage.crons(0).lastRun.indexOf('Last:') === 0, 'Shows last run time of cron build');
```

Comment:
I think there might be a cleaner way to do this, similar to how it's done in the line above.  
```
assert.ok(settingsPage.crons(0).lastRun, 'Last:', 'Shows last run time of cron build');
```